Update README.md (#1)
psycho-ir/mr-hashemi: README.md

Mr. Hashemi is a *National* programming language.
We hope Mr. Hashemi get enough attention to make bilions of tomans out of this project.  

**example:**
```java
bebin main() {
  i = 0;

>   ta (i<1) bood{

Reviewer: ```suggestion
  ta (i<1) bood {
```

    bechap("adad bede");
    n = adadBekhoon();
    derakht(n);
  }
}

bebin derakht(n){
  i = 0;
  matn = "";

  ta (i <n) bood {
    matn = matn + "*";
    i = i + 1;
    bechap(matn);
  }

  j = 0;
  ta (j<4) bood {
    bechap("*");
    j = j+1;
  }
}
```  
  
## Basic syntax:  

> ### Methods:  

Reviewer: ```suggestion
### Functions:  
```

> - Declare a method by keyword `bebin` before method name.  

Reviewer: ```suggestion
- Declare a function by keyword `bebin` before function name.  
```

> - Program starts from main method.  

Reviewer: ```suggestion
- Program starts from main function.  
```

> - to call a method simply call it.  

Reviewer: ```suggestion
- to call a function simply call it.  
```

> in example above the methods `main` and `derakht` are defined and method `derakht` is called inside main.

Reviewer: ```suggestion
in example above the functions `main` and `derakht` are defined and function `derakht` is called inside main.
```

### Loops:  
The usual `while (condition) {code}` loop is defined as `ta (shart) bood {code}` .  
### condition:  

> The usual `if (condition) {code}` loop is defined as `age (shart) {code}` .  

Reviewer: ```suggestion
The usual `if (condition) {code}` conditions definition is defined as `age (shart) bood {code}` .  
```
